### PR TITLE
Add support for MINBATCHTIMEOUT

### PR DESCRIPTION
**Pull request by billschereriii**
Paralleling the implementation of support for min_batch_size

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -19,16 +19,21 @@ To be released at some future point in time
...
+- Added support for MINBATCHTIMEOUT in model execution, which caps the delay
+  waiting for a minimium number of model execution operations to accumulate
+  before executing them as a batch (PR387_)
 - RedisAI 1.2.5 is not supported anymore. The only RedisAI version
   is now 1.2.7. Since the officially released RedisAI 1.2.7 has a
   bug which breaks the build process on Mac OSX, it was decided to
   use commit 634916c_ from RedisAI's GitHub repository, where such
   bug has been fixed. This applies to all operating systems. (PR383_)
 
+  .. _PR384: https://github.com/CrayLabs/SmartSim/pull/387
```
Comment (al-rigazzi) [inline doc/changelog.rst:36]: `_PR384` -> `_PR387`

**Review by al-rigazzi — CHANGES_REQUESTED** [review]
Everything looks good except for a tiny typo in the changelog. Otherwise LGTM!

**Review by ashao — APPROVED** [review]
Looks good. Thanks!

**Review by al-rigazzi — APPROVED** [review]
LGTM! Thanks for addressing this so quickly!

**Pull request merged**